sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client_options.hpp

```diff
@@ -9,28 +9,62 @@
 
 #pragma once
 
-#include <azure/keyvault/common/client_options.hpp>
+#include <azure/core/internal/client_options.hpp>
 
 #include "azure/keyvault/keys/key_vault_key.hpp"
 
 namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
 
+  enum class ServiceVersion
+  {
+    /**
+     * @brief Use to send request to the 7.0 version of Key Vault service.
+     *
+     */
+    V7_0,
```

Comment:
Does this syntax and naming pattern align with other the other SDKs?

Reply:
Yes, Net and Java